nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -163,7 +163,11 @@
 - module-name: 'arcade' # checksum: 3f9f0b7d
   data-files:
     dirs:
-      - 'resources'
+      - 'resources\system'
+  anti-bloat:
+    - no-auto-follow:
+        'examples': 'ignore'
+        'experimental': 'ignore'
```

Comment:
Just guessing here, but I assume experimental feature usage is rare, in that case, you should tell the user about it, and inform them that experimental features are not going to be available by not putting 'ignore' there.

An example of this would be

```
- module-name: 'joblib.externals.cloudpickle.cloudpickle' # checksum: 1d738df0
  anti-bloat:
    - no-auto-follow:
        'tornado': 'can break tornado integration of joblib'
```

Although, it probably should say. "Experimental features of arcade will not be available unless you do '--include-module=acade.experimental'" so the new user knows about this and also knows the remedy. I agree these texts are not generally as helpful as they ought to be.

Reply:
That makes a lot of sense. I will also update Arcade's docs with the same info. Thank you.

Reply:
`FATAL: Error, cannot add data file with 'arcade\resources/system\gamecontrollerdb.txt' path, as 'contains illegal character '/''`